src/features/Classes/InstructorCard/index.jsx

```diff
@@ -65,21 +61,28 @@ const InstructorCard = () => {
       </div>
       <div className="separator" />
       <div className="instructor-details">
-        <h4 className="text-color text-uppercase mb-3 h5">Instructor</h4>
-        <div className="d-flex align-items-center">
-          <img
-            src={instructors[0]?.instructorImage ? instructors[0]?.instructorImage : instructorDefaultImage}
-            alt="Instructor profile"
-            className="instructor-image"
-          />
-          <span className="text-capitalize ml-3 text-black font-weight-bold text-truncate w-50 d-block" title={classInfo?.instructors[0]}>
-            {classInfo?.instructors[0]}
-          </span>
+        <h4 className="text-color text-uppercase mb-3 h5">Instructor{instructors.length > 1 && 's'}</h4>
+        <div className="d-flex align-items-center flex-wrap">
+          {
+            classInfo?.instructors?.slice(0, 3)?.map((instructor) => {
```

Comment:
For three instructors, the section doesn't look good
![Screenshot from 2024-06-03 14-06-58](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/36944773/00423fa2-c48c-45c6-b7f7-ac7e10662985)

Reply:
Fixed